package.json

```diff
@@ -20,8 +20,13 @@
   },
   "homepage": "https://github.com/hislordshipprof/Todo-list#readme",
   "devDependencies": {
+    "@babel/core": "^7.18.6",
+    "@babel/preset-env": "^7.18.6",
+    "babel-jest": "^28.1.2",
```

Comment:
- [ ] I strongly recommend adding     "test": "jest", in scripts, so you can run rests by `npm run test` 👍🏻